mobsfscan/rules/patterns/ios/swift/best_practices.yaml

```diff
@@ -68,9 +68,13 @@
     owasp-mobile: m1
     reference: https://github.com/MobSF/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#app-extensions
 - id: ios_keyboard_cache
-  message: This app does not disable Keyboard cache. It must be disabled for all sensitive data inputs.
+  message: This app may disable Keyboard cache. It must be disabled for all sensitive data inputs.
   input_case: exact
-  pattern: \.autocorrectionType = \.no
+  pattern:
+    - \.autocorrectionType\s*=\s*\.no
+    - \.autocorrectionType\s*=\s*\.No
+    - UITextAutocorrectionTypeNo
+    - textField\.disableAutocorrection\s*\(\s*true\s*\)
```

Comment:
Can `textField` be omitted?
`disableautocorrection` is an instance method https://developer.apple.com/documentation/swiftui/text/disableautocorrection(_:)